### ISSUE TITLE
[ZeroNode related] Should zero node support aws instance with EBS-only without additional ebs disks?

### ISSUE THREAD

**Issue by aleksbykov** [scylladb/scylladb#21321]
Zero nodes could use slow aws instances because zero nodes are not store any user data. Currently we can run Scylla on slower instances like (r6?, c6?) with additional ebs disks to build raid0. 
Should scylla zero token be started on slower instances without any extra ebs storage?

**Comment by mykaul** [comment]
Im not sure an issue is the best venue to ask clarifying questions on a feature.

**Comment by kbr-scylla** [comment]
Zero token nodes need persistent storage like regular nodes. They store group 0 state (schema, topology metadata and so on).
But the amount of storage required should be small compared to regular nodes. So single disk should be enough (how large is it?), raid0 should not be necessary.

**Comment by patjed41** [comment]
> But the amount of storage required should be small compared to regular nodes.  So single disk should be enough (how large is it?), raid0 should not be necessary.

Yes, a single disk should be enough, but we have already seen surprises like `cdc_generations_v3` exceeding 100 MB in #17545 (this problem has been fixed).

Also, I don't know how Scylla behaves with less storage even if this storage should suffice with no user data. Hopefully, nothing random breaks.

So, I think we should just check that zero-token nodes work without any extra storage. This issue could be about this check instead of being a question.

**Comment by mykaul** [comment]
> > But the amount of storage required should be small compared to regular nodes.  So single disk should be enough (how large is it?), raid0 should not be necessary.
> 
> Yes, a single disk should be enough, but we have already seen surprises like `cdc_generations_v3` exceeding 100 MB in [#17545](https://github.com/scylladb/scylladb/issues/17545) (this problem has been fixed).

100MB is still very small.

> 
> Also, I don't know how Scylla behaves with less storage even if this storage should suffice with no user data. Hopefully, nothing random breaks.
> 
> So, I think we should just check that zero-token nodes work without any extra storage. This issue could be about this check instead of being a question.


I agree, I think even using the root disk is reasonable, for cost perspective (and an ARM machine).
